backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -41,11 +41,12 @@ public FutureExperimentRunDAO(Executor executor, FutureJdbi jdbi, UAC uac) {
     this.jdbi = jdbi;
     this.uac = uac;
 
-    attributeHandler = new AttributeHandler(executor, jdbi);
-    hyperparametersHandler = new KeyValueHandler(executor, jdbi, "hyperparameters");
-    metricsHandler = new KeyValueHandler(executor, jdbi, "metrics");
+    attributeHandler = new AttributeHandler(executor, jdbi, "ExperimentRunEntity");
```

Comment:
We have a constants for the entities name as well as fields name like `hyperparameters` Could we use that??